src/utils/media-utils.js

```diff
@@ -27,6 +28,7 @@ export function analyzeMediaUrl(url) {
     return {
       sanitizedUrl: `https://www.youtube.com/watch?v=${encodeURIComponent(videoId)}`,
       isYoutube: true,
+      youtubeVideoId: videoId,
```

Comment:
I'm only partial to this, but what would you think of removing `isYoutube` in this case and having `youtubeVideoId` as the sole indicator?

Reply:
I thought the same and was not sure if it's worth it, but as you have the same opinion, I'll go for it.

Reply:
Removed (and found out that it was not used anymore anyways).